BacktraceScreen.h

```diff
@@ -0,0 +1,46 @@
+#ifndef HEADER_BacktraceScreen
+#define HEADER_BacktraceScreen
+/*
+htop - Filename.h
+(C) 2021 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
+
+#if (defined(HTOP_LINUX) && defined(HAVE_LIBUNWIND_PTRACE))
+
+#include <stddef.h>
+
+#include "Panel.h"
+#include "Process.h"
+
+typedef struct BacktracePanel_ {
+   Panel super;
+   const Process* process;
+   char* error;
+} BacktracePanel;
+
+typedef struct Frame_ {
+   Object super;
+   int index;
+   size_t address;
+   size_t offset;
+   char* functionName;
+   bool isSignalFrame;
+
+   bool isError;
```

Comment:
I didn't quite get the idea why you need this "isError" boolean flag. Is it not enough to check whether the "error" pointer is NULL?

Reply:
I can't entirely agree with what I did about error management. I mean, the structure `Frame` should contain only data associated with the frame. And It doesn't make sense for me to have a field `error `representing a general error on a `frame`. I have to try other ways to show an error.